### PR TITLE
[zh] Fix incorrect description for Service in endpoint-slices.md

### PR DESCRIPTION
**Pull request by fenggw-fnst**
Signed-off-by: Guangwen Feng <fenggw-fnst@fujitsu.com>

Inspired by [this comment](https://github.com/kubernetes/website/pull/33823#discussion_r876652479) and by the way modified the entire file.
/assign @tengqm
PTAL, thanks.

### CODE REVIEW

File: content/zh/docs/concepts/services-networking/endpoint-slices.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ significant amounts of network traffic and processing when Endpoints changed.
 EndpointSlices help you mitigate those issues as well as provide an extensible
 platform for additional features such as topological routing.
 -->
-由于任一服务的所有网络端点都保存在同一个 Endpoints 资源中，这类资源可能变得
+由于任一 Service 的所有网络端点都保存在同一个 Endpoints 资源中，这类资源可能变得
```
Comment (howieyuen) [inline content/zh/docs/concepts/services-networking/endpoint-slices.md:55]: 这里可以考虑按照标点符合折行，这样渲染出来的结果不会有多余的空格

**Review by howieyuen — CHANGES_REQUESTED** [review]

File: content/zh/docs/concepts/services-networking/endpoint-slices.md (hunk 1 above)
Comment (fenggw-fnst) [reply content/zh/docs/concepts/services-networking/endpoint-slices.md]: 谢谢 review，已经根据建议进行了全文修改以避免多余空格。

**Pull request merged**